SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -868,6 +868,9 @@ protected SwrveConversation _getConversationForEvent(String event) {
             }
             synchronized (campaigns) {
                 List<SwrveConversation> availableConversations = new ArrayList<SwrveConversation>();
```

Comment:
Its not, one list has the highest priority messages / conversations where we will pick randomly, the other has all of the campaign messages / conversations that were ready to be shown, but were not because the other was chosen.